crates/hir-def/src/import_map.rs

```diff
@@ -167,11 +167,7 @@ fn collect_import_map(db: &dyn DefDatabase, krate: CrateId) -> ImportMap {
 
         let visible_items = mod_data.scope.entries().filter_map(|(name, per_ns)| {
             let per_ns = per_ns.filter_visibility(|vis| vis == Visibility::Public);
-            if per_ns.is_none() {
-                None
-            } else {
-                Some((name, per_ns))
-            }
+            if per_ns.is_none() { None } else { Some((name, per_ns)) }
```

Comment:
This is the result of some nightly only options rust-lang/rust uses: https://github.com/rust-lang/rust/blob/2f320a224e827b400be25966755a621779f797cc/rustfmt.toml#L2-L4 Stable rustfmt without those options as used in this repo will revert these changes. You will need to do one of:

* Copy the options to a rustfmt.toml in this repo and switch to nightly rustfmt
* Ignore rust-analyzer for rustfmt in rust-lang/rust
* Explicitly use the stable defaults of those options in rustfmt.toml in this repo and hope rustfmt will pick them in favor of the global rustfmt.toml of rust-lang/rust

Reply:
That one's my bad, I had "format on save" set up when doing those edits. I'll revert them in this PR.